tests/plugins/find/find.js

```diff
@@ -263,6 +263,41 @@ bender.test( {
 		} );
 	},
 
+	// (#5061)
+	'test replace one of occurences of phrase with phrase with several spaces inside': function() {
+		var bot = this.editorBot;
+
+		bot.setHtmlWithSelection( '<p>replace me</p>' );
+
+		bot.dialog( 'replace', function( dialog ) {
+			dialog.setValueOf( 'replace', 'txtFindReplace', 'replace me' );
+			dialog.setValueOf( 'replace', 'txtReplace', 'foo   bar' );
+			dialog.getContentElement( 'replace', 'btnFindReplace' ).click();
+			dialog.getContentElement( 'replace', 'btnFindReplace' ).click();
+
+			assert.areSame( '<p><span title="highlight">foo&nbsp; &nbsp;bar</span></p>', bot.getData( true ) );
+
+			dialog.getButton( 'cancel' ).click();
+		} );
+	},
+
+	// (#5061)
+	'test replace all occurences of phrase with phrase with several spaces inside': function() {
+		var bot = this.editorBot;
+
+		bot.setHtmlWithSelection( '<p>[replace me]</p><p>replace me</p>' );
+
+		bot.dialog( 'replace', function( dialog ) {
+			dialog.setValueOf( 'replace', 'txtReplace', 'foo   bar' );
+			dialog.getContentElement( 'replace', 'btnReplaceAll' ).click();
+			dialog.getButton( 'cancel' ).click();
```

Comment:
We can remove this line because we hide dialog after assertion. ( line 297 )

Reply:
Actually, I've moved it before the assertion in both tests. Otherwise, if the assertion failed, the dialog wouldn't be closed.